src/main.cpp

```
@@ -64,14 +64,23 @@ int usage()
+static void append_pg_cl_sanitized(kstring_t *pg, const char *arg)
 {
-    kputc(' ', pg);
     for (const char *c = arg; *c != '\0'; ++c) {
         kputc((*c == '\t' || *c == '\n' || *c == '\r') ? ' ' : *c, pg);
     }
 }
 
+// A single space-separated argv token (argv[1..]). argv[0] is appended by
+// append_pg_cl_sanitized directly (no leading space, right after "CL:").
+static void append_pg_cl_arg(kstring_t *pg, const char *arg)
+{
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)